content/zh-tw/cluster.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 叢集
+status: Completed
+category: 概念
+tags: ["基礎架構", "架構", ""]
+---
+
+## 是什麼
+
+一座叢集是指一組由計算機或應用程式為實現共同目標而協同工作。
+在雲原生計算的背景之下，這個詞彙通常用於 Kubernetes。
```

Comment:
original
```
In the context of cloud native computing, the term is most often applied to [Kubernetes](/kubernetes/). 
```
```suggestion
在雲原生計算的背景之下，該術語最常用於 [Kubernetes](/zh-tw/kubernetes/)。
```

Reply:
![image](https://github.com/cncf/glossary/assets/64598482/57a9aff1-1735-4981-abf2-12b9916364ec)

Links are used in the original text, but not in Zh-tw.